test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests/RestoreCommandTests.cs

```diff
@@ -1984,12 +1984,22 @@ public async Task RestoreCommand_DowngradeIsNotErrorWhen_DowngradedByCentralDire
             }
         }
 
-        [Fact]
-        public async Task RestoreCommand_CentralVersion_ErrorWhenFloatingCentralVersions()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+
+        public async Task RestoreCommand_CentralVersion_ErrorWhenFloatingCentralVersionsAreDisabled(bool enabled)
```

Comment:
I'm going to leave this as-is for now since its an existing test that I'm editing.  I want to be able to see history of the test in case it starts failing intermittently after this change.  All of the test names should be updated later in another PR if we get to it.

Reply:
A follow-up PR to address the whole thing is generally fine, but I don't follow your comment about seeing test case history. You edited the test name in this PR, so how is history being affected by editing it again before merging?

Reply:
Looks like you put the original test name back. Ok, makes sense.